main.py

```diff
@@ -1029,6 +1042,9 @@ def run_analysis(self) -> None:
             if self.zap_options:
                 command = f"{command} {Constants.ZAP_OPTIONS} \"{self.zap_options}\""
 
+            if self.other_options:
+                command = f"{command} {self.zap_options}"
```

Comment:
self.other options instead of self.zap_options

Reply:
fixed